src/main/scala/org/broadinstitute/dsde/workbench/sam/model/SamModel.scala

```diff
@@ -238,6 +239,12 @@ object BasicWorkbenchGroup {
 @Lenses final case class ManagedGroupAndRole(groupName: WorkbenchGroupName, role: MangedGroupRoleName)
 @Lenses final case class ManagedGroupMembershipEntry(groupName: ResourceId, role: ResourceRoleName, groupEmail: WorkbenchEmail)
 @Lenses final case class ManagedGroupAccessInstructions(value: String) extends ValueObject
+@Lenses final case class ManagedGroupSupportSummary(
+    groupName: WorkbenchGroupName,
+    email: WorkbenchEmail,
+    authDomainFor: Set[FullyQualifiedResourceId],
+    parentGroups: Set[WorkbenchGroupName]
+)
```

Comment:
We are starting to try to break out parts of this file into smaller ones, maybe it is worth making a new ManagedGroup file under `model/api`?

Reply:
https://github.com/broadinstitute/sam/blob/develop/src/main/scala/org/broadinstitute/dsde/workbench/sam/model/api/SamUser.scala